examples/boilerplate/teapot/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Teapot • A-Frame</title>
+    <meta name="description" content="Teapot • A-Frame" />
+    <script type="importmap">
+      {
+        "imports": {
+          "aframe": "../../../dist/aframe-master.module.min.js",
+          "three": "https://cdn.jsdelivr.net/npm/super-three@0.172.0/build/three.module.js",
```

Comment:
Problem is that will have to update this in the future since it’s locked to r172. Is it possible to have an example without a hardcoded version?

Reply:
It could be an option to tag the super-three versions with the A-Frame version they are used in, so you'd be able to use `super-three@aframe-1.7.0`. For master we could consider pointing it to `node_modules` instead of using a CDN? That way when checking out older revisions, running `npm install` ensures that the right super-three version is referenced.

Reply:
yeah. pointing to node_modules sounds fine to me. trying to avoid small things that will eventually break and hard to keep track of over time.

Reply:
Did you see my changes to the release script? There is no need to update manually, the script will update the super-three version in examples and docs similar to the aframe version.
But indeed for master that should reference the current version in node_modules. I just added an additional directory path to the webpack dev server config, and the preghpages script rewrite the url with the cdn.